medallion/test/data/initialize_mongodb.py

```diff
@@ -306,6 +314,20 @@ def reset_db(url="mongodb://root:example@localhost:27017/"):
             "spec_version": "2.1",
             "type": "malware",
             "_collection_id": "91a7b528-80eb-42ed-a74d-c6fbd5a26116",
+        },
+        {
+            "created": datetime_to_float(string_to_datetime("2017-01-27T13:49:53.997Z")),
+            "description": "Poison Ivy",
+            "id": "malware--c0931cc6-c75e-47e5-9036-78fabc95d4ec",
+            "is_family": True,
+            "malware_types": [
+                "remote-access-trojan",
+            ],
+            "modified": datetime_to_float(string_to_datetime("2018-02-23T18:30:00.000Z")),
+            "name": "Poison Ivy",
+            "spec_version": "2.0",
```

Comment:
This property can be removed completely. This will better represent a 2.0 object.